### PR TITLE
Support XPU export

### PR DESCRIPTION
**Pull request by Kaihui-intel**
Support XPU export

### CODE REVIEW

File: auto_round/export/export_to_itrex/export.py [hunk 1]
```diff
@@ -145,10 +201,11 @@ def pack_model(
             scale_dtype=scale_dtype,
             zp=zp is not None,
             bias=m.bias is not None,
-            device=device,
-            use_optimum_format=True,
+            device="cuda" if device == "xpu" else device,
```
Comment (WeiweiZhang1) [inline auto_round/export/export_to_itrex/export.py:204]: It might be a bit obscure here, can you change it? for example, add a temporary variable quant_device

File: auto_round/export/export_to_itrex/export.py [hunk 2]
```diff
@@ -127,6 +180,9 @@ def pack_model(
         if not isinstance(scale, torch.Tensor):
             scale = torch.tensor(scale, dtype=convert_dtype)
             zp = torch.tensor(zp, dtype=torch.int32)
+            if device == "xpu":
+                scale = torch.tensor(v["scale"], dtype=torch.float32)
```
Comment (WeiweiZhang1) [inline auto_round/export/export_to_itrex/export.py:184]: better add a log here, if convert_dtype != torch.float32 there maybe lead to loss of acc
